addon/styles/_frost-subcontext-actions.scss

```diff
@@ -0,0 +1,85 @@
+$frost-subcontext-actions-button-bg: $frost-color-white;
+$frost-subcontext-actions-button-shadow: $frost-color-grey-6;
+$frost-subcontext-actions-button-icon: $frost-color-grey-4;
+$frost-subcontext-actions-button-icon-selected: $frost-color-blue-1;
+$frost-subcontext-actions-button-icon-active: $frost-color-blue-1;
+$frost-subcontext-actions-label-background: rgba(0, 0, 0, .65);
+@keyframes frost-subcontext-actions-label-animation-out {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+
+}
+
+@keyframes frost-subcontext-actions-label-animation-in {
+  to {
+    // right: 0%;
+    opacity: 0;
+    visibility: hidden;
+  }
+
+  from {
+    // right: 170%;
+    opacity: 1;
+    visibility: visible;
+  }
+
+}
+
+.frost-subcontext-actions {
+  width: 60px;
+  height: 100%;
+  margin-left: auto;
```

Comment:
I'd remove the margin-left - let the consumer place it (this came from the original - I should have removed it)